src/Turtle/Prelude.hs

```diff
@@ -1331,6 +1332,28 @@ fork io = using (managed (withAsync io))
 wait :: MonadIO io => Async a -> io a
 wait a = liftIO (Control.Concurrent.Async.wait a)
 
+{-| Run a list of actions in parallel using fork and wait
+
+>>> sh . using $ parallel $ fmap (\i -> Turtle.shells "sleep 2 && date" empty) [1..10]
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+Sat Nov 19 17:26:52 CET 2016
+
+-}
+parallel :: MonadManaged managed => [IO ()] -> managed ()
```

Comment:
But can we do better?? 😉 

> If I had more time, I would have written a shorter letter.

\- [Blaise Pascal](http://quoteinvestigator.com/2012/04/28/shorter-letter/)